components/frontend/src/utils.js

```diff
@@ -322,6 +322,18 @@ export function scaled_number(number) {
     return (number / Math.pow(1000, exponent)).toFixed(0) + scale[exponent]
 }
 
+export function formatMetricValue(scale, value) {
+    // metric value may be a string (version number or percentage), float, or integer
+    if (value === "?") {
+        return value
+    }
+    if (scale === "count") {
+        const number = Number(value)
+        return number.toLocaleString(undefined, { useGrouping: true })
```

Comment:
The thousand separators comes from LocaleString builtin? Perhaps then it's better to rename the change to "apply locale to digits", and is this something we can add tests for?

Reply:
I used a more generic name because I think there's more metric value formatting that could be moved to this method, like adding a percentage symbol. I'll see what can easily be moved here.